.github/pull_request_template.md

```diff
@@ -2,22 +2,23 @@
 
 **What is this about:**
 
-**What is included in the PR:** 
 
 **How does someone test / validate:** 
 
 ## Quality Checklist
 
 - [ ] **Linked issue:** #xxx
-- [ ] **Communication:** I've discussed this with core contributors in the issue. 
```

Comment:
I want this, this was one of the reasons why we added this.  It is a reminder to the person that all parties agreed on what was planned to be done.